api/activity.go

```diff
@@ -241,11 +242,12 @@ type ActivityProjectDatabaseTransferPayload struct {
 // ActivitySQLEditorQueryPayload is the API message payloads for the executed query info.
 type ActivitySQLEditorQueryPayload struct {
 	// Used by activity table to display info without paying the join cost
-	Statement    string `json:"statement"`
-	DurationNs   int64  `json:"durationNs"`
-	InstanceName string `json:"instanceName"`
-	DatabaseName string `json:"databaseName"`
-	Error        string `json:"error"`
+	Statement    string             `json:"statement"`
+	DurationNs   int64              `json:"durationNs"`
+	InstanceName string             `json:"instanceName"`
+	DatabaseName string             `json:"databaseName"`
+	Error        string             `json:"error"`
+	Advice       advisor.AdviceList `json:"advice"`
```

Comment:
`AdviceList` may be better.
```suggestion
	AdviceList       advisor.AdviceList `json:"adviceList"`
```